packages/backend/package.json

```diff
@@ -12,7 +12,8 @@
     "migrate:create": "pnpm prisma migrate dev --create-only",
     "dev": "nodemon",
     "lint": "eslint --ext .ts src",
-    "build": "tsc && resolve-tspaths && pnpm generate --nexusTypegen",
+    "migrate:deploy": "prisma migrate deploy",
+    "build": "pnpm install --prod=false && pnpm generate && tsc && resolve-tspaths && pnpm migrate:deploy",
```

Comment:
Do we want to have --prod=false? It makes sense for bootstrapping a project, but I wonder if it will get missed when deploying to prod and bloat the build?

Reply:
I'm pretty sure I added this flag on a project recently to get our backend Render builds working. We were running into an issue because we'd listed `resolve-tspaths` as a dev dependency, but it's required for our backend to build.

I don't think we really need to be concerned about the size of our backend builds, but likely a better long-term solution would be to move `resolve-tspaths` into the dependencies list.

@dgillen27 were you running into a similar issue?